src/network/netdev/tunnel.c

```diff
@@ -435,20 +435,26 @@ static int netdev_tunnel_verify(NetDev *netdev, const char *filename) {
 
         assert(t);
 
-        if (IN_SET(netdev->kind, NETDEV_KIND_VTI, NETDEV_KIND_IPIP, NETDEV_KIND_SIT, NETDEV_KIND_GRE, NETDEV_KIND_GRETAP) &&
-            t->family != AF_INET)
-                return log_netdev_error_errno(netdev, SYNTHETIC_ERRNO(EINVAL),
-                                              "vti/ipip/sit/gre tunnel without a local/remote IPv4 address configured in %s. Ignoring", filename);
+        if (IN_SET(netdev->kind, NETDEV_KIND_VTI, NETDEV_KIND_IPIP, NETDEV_KIND_SIT, NETDEV_KIND_GRE)) {
+                if (t->family == AF_UNSPEC)
+                        t->family = AF_INET;
+                if (t->family != AF_INET)
```

Comment:
not that it matters at all, but I'd have prefixed this line with an `else`, since there's no need to check what we just assigned...